Fix the markup on the "mail" part of the manpage (Closes: #300644)


---
 logrotate.8 |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: logrotate-3.7.1/logrotate.8
===================================================================
--- logrotate-3.7.1.orig/logrotate.8
+++ logrotate-3.7.1/logrotate.8
@@ -45,7 +45,7 @@ by hand, as the new files will be create
 correctly.
 
 .TP
-\fB-m\R, \B-\-mail <command>\fR
+\fB-m\fR, \fB-\-mail <command>\fR
 Tells \fBlogrotate\fR which command to use when mailing logs. This
 command should accept two arguments: 1) the subject of the message, and
 2) the recipient. The command must then read a message on standard input
